docs/install.rst

```diff
@@ -35,6 +35,25 @@ following line in your ``.zshrc`` after activating Antigen::
 
    antigen bundle nojhan/liquidprompt
 
+Installation via Zinit
+----------------------
+
+To install via `Zinit <https://github.com/zdharma/zinit>`_, simply add the
+following line in your ``.zshrc`` after activating Zinit::
+
+   zinit load nojhan/liquidprompt
```

Comment:
I just remembered, mentioned in https://github.com/nojhan/liquidprompt/issues/646#issuecomment-765661569, we have a `stable` branch, tracking stable releases (since `master` is the development branch). The docs were updated to recommend users install the stable branch, so I would like this too as well.
```suggestion
   zinit load ver"stable" nojhan/liquidprompt
   
Or, if you want to use the development (non-stable) branch::

   zinit load nojhan/liquidprompt
```
It seems to be that simple with Zinit?